logging/tests/unit/handlers/test__helpers.py

```diff
@@ -63,6 +64,57 @@ def test_valid_context_header(self):
         self.assertEqual(trace_id, expected_trace_id)
 
 
+# webapp2 has not been ported to python3 yet, so do not try to test its
+# functionality there.
+if sys.version_info[0] == 2:
```

Comment:
Nice!  I figured I wasn't doing it in the best way.  Though I do think that the interpreter still tries to byte-compile the test in this case?  Which means that in python3 it will still try to import webapp2 and break.  I guess if we do your other suggestion we'll catch the problem that way.